.github/workflows/backport.yml

```
@@ -0,0 +1,22 @@
+---
+name: Backport Assistant Runner
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.2.2
```

[Looks like](https://hub.docker.com/r/hashicorpdev/backport-assistant/tags) the latest version is `0.2.5`

```suggestion
    container: hashicorpdev/backport-assistant:0.2.5
```